### PR TITLE
Create 1776 release note

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Create 1776 release note

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- [Add new Order - An exception is displayed when ordering a product with text customization - the label contains special expression](https://github.com/PrestaShop/PrestaShop/issues/25194)
+
+
+**Install**
+- [Cannot create an alternative currency after an upgrade](https://github.com/PrestaShop/PrestaShop/issues/24871)
+
+**Front-office**
+- [Brands are shown wrongly in the FO](https://github.com/PrestaShop/PrestaShop/issues/25214)
+- ['Total available for each user' in cart rule not considered if the customer is signed in from the checkout page](https://github.com/PrestaShop/PrestaShop/issues/25203)
+- [Smarty template isCached function bug](https://github.com/PrestaShop/PrestaShop/issues/24235) - CRITICAL
```
Comment (matks) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]: Suggestion: move critical bugfixes into a dedicated section, and provide context about why they are critical (maybe remind our labels definitions)

Comment (marionf) [reply news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]: Ok, done 👍

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+## Critical fixes
+
+The bug affects critical functionality or critical data and there is no workaround to avoid it. A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Two critical bugs have been fixed in 1.7.7.6:
+
+- [Orders page - an exception is displayed when deleting a customer or a guest](https://github.com/PrestaShop/PrestaShop/issues/25319)
+- [Smarty template isCached function bug](https://github.com/PrestaShop/PrestaShop/issues/24235)
```
Comment (matks) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
- [Smarty template caching Module::isCached() faulty causes performance loss](https://github.com/PrestaShop/PrestaShop/issues/24235)
```
I improved the title to make it easier to understand what this is about

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Main fixes
```
Comment (matks) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]: "Main fixes" seems to indicate there are "main" fixes and "other" fixes. But there is no other fixes, if I am not wrong ?

Then maybe we can simply write
```suggestion
## Bug fixes list
```

Comment (marionf) [reply news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]: The others are the critical ones

**Review by matks — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @marionf

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.7.6"
+subtitle: "Maintenance version of the 1.7.7.x branch"
+date:   2021-08-06 12:00:00
```
Comment (eternoendless) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
date:   2021-08-09
```

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+![1.7.7.6 is available!](/assets/images/2021/08/build_1.7.7.6.png)
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don't forget to backup before.
+
+## Critical fixes
+
+The bug affects critical functionality or critical data and there is no workaround to avoid it. A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
```
Comment (eternoendless) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]: I think this part can be removed
```suggestion
```

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md (hunk 3 above)
Comment (eternoendless) [reply news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
## Other bug fixes
```

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+Two critical bugs have been fixed in 1.7.7.6:
+
+- [Orders page - an exception is displayed when deleting a customer or a guest](https://github.com/PrestaShop/PrestaShop/issues/25319)
+- [Smarty template caching Module::isCached() faulty causes performance loss](https://github.com/PrestaShop/PrestaShop/issues/24235)
+
+## Main fixes
+
+**Back-office**
```
Comment (matthieu-rolland) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
**Back-office**

    - [Fixed selector for fetching if the tax is included](https://github.com/PrestaShop/PrestaShop/pull/25489)
    - [An exception is displayed when we click on sign out two times](https://github.com/PrestaShop/PrestaShop/pull/25495)
```

File: news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+- [Orders page - an exception is displayed when deleting a customer or a guest](https://github.com/PrestaShop/PrestaShop/issues/25319)
+- [Smarty template caching Module::isCached() faulty causes performance loss](https://github.com/PrestaShop/PrestaShop/issues/24235)
+
+## Other bug fixes
+
+**Back-office**
+
+    - [Fixed selector for fetching if the tax is included](https://github.com/PrestaShop/PrestaShop/pull/25489)
+    - [An exception is displayed when we click on sign out two times](https://github.com/PrestaShop/PrestaShop/pull/25495)
+- [Preview order - Invoice details is missing the address mail of the customer](https://github.com/PrestaShop/PrestaShop/issues/24482)
```
Comment (matthieu-rolland) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
- [Fixed selector for fetching if the tax is included](https://github.com/PrestaShop/PrestaShop/pull/25489)
- [An exception is displayed when we click on sign out two times](https://github.com/PrestaShop/PrestaShop/pull/25495)
- [Preview order - Invoice details is missing the address mail of the customer](https://github.com/PrestaShop/PrestaShop/issues/24482)
```

Comment (matthieu-rolland) [inline news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md]:
```suggestion
- [Fixed selector for fetching if the tax is included](https://github.com/PrestaShop/PrestaShop/pull/25489)
- [An exception is displayed when we click on sign out two times](https://github.com/PrestaShop/PrestaShop/pull/25495)
- [Preview order - Invoice details is missing the address mail of the customer](https://github.com/PrestaShop/PrestaShop/issues/24482)
```

**Review by eternoendless — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matthieu-rolland & @marionf

**Pull request merged**